app/src/main/java/com/iic/shopingo/ui/request_flow/views/CreateRequestItemListView.java

```diff
@@ -108,4 +123,8 @@ public void onItemEdited(View view, String value) {
     }
   }
 
+  public interface OnRequestItemListChanged {
+    public void onItemListChanged(View view);
```

Comment:
Theoretically, if you had a number of them in an activity, you could use it to get its items, but I guess we'll add that in if needed.